spectrocloud/workspace_backup.go

```diff
@@ -71,6 +71,6 @@ func getExtraFields(d *schema.ResourceData) (bool, []string) {
 	return include_all_clusters, nil
 }
 
-func detachWorkspaceBackupPolicy(c *client.V1Client) error {
-	return errors.New("Not implemented") // not implemented
-}
+// func detachWorkspaceBackupPolicy(c *client.V1Client) error {
```

Comment:
let's keep it as feature will be implemented at some point.

Reply:
sure! leaving it commented out though to avoid lint errors